canary/src/main/java/io/littlehorse/canary/config/KafkaConfig.java

```diff
@@ -2,14 +2,14 @@
 
 import java.util.Map;
 import java.util.stream.Collectors;
-import org.apache.kafka.clients.consumer.ConsumerConfig;
 
-public class KafkaConsumerConfig implements Config {
+public class KafkaConfig implements Config {
     private final Map<String, Object> configs;
 
-    public KafkaConsumerConfig(final Map<String, Object> configs) {
+    public KafkaConfig(final Map<String, Object> configs) {
         this.configs = configs.entrySet().stream()
-                .filter(entry -> ConsumerConfig.configNames().contains(entry.getKey()))
+                .filter(entry -> entry.getKey().startsWith("kafka."))
```

Comment:
I realized that there is a better way to do this. Rather than deleting the other files and using the `kafka.` prefix, we can make the `.filter()` also check for:
- `ConsumerConfig`
- `CommonClientConfig`
- `SslConfig`

It was ignoring `CommonClientConfig` and `SslConfig`. I think that smaller solution is much better.

Reply:
Hmm, this might be something we optimize for later.